tests/JsonTest.php

```diff
@@ -65,6 +66,16 @@ public function testDecodeString1()
     }
 
 
+    /**
+     * @expectedException duncan3dc\Serial\Exceptions\JsonException
+     * @expectedExceptionMessage JSON Error: Syntax error
```

Comment:
Please can you change this to use this methods (`expectException()`, `expectExceptionMessage()`)?

Reply:
As you wish.